package.json

```diff
@@ -14,13 +14,14 @@
   "devDependencies": {
     "tslint": "^5.4.3",
     "typedoc": "^0.7.1",
-    "typescript": "~2.3.4",
+    "typescript": "~2.5.2",
```

Comment:
the peerDependency should be updated, then this is ready to go

Reply:
Should I do this or do you want to?

Reply:
Ping me if I should

Reply:
I don't care :D

Reply:
@eseliger please do, I'm on mobile for a while.